app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", int, float]) -> "Distance":
+        if isinstance(other, Distance):
+            length = self.km + other.km
+        else:
+            length = self.km + other
+        return Distance(length)
+
+    def __iadd__(self, other: Union["Distance", int, float]) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, num: int) -> "Distance":
+        length = self.km * num
+        return Distance(length)
+
+    def __truediv__(self, num: int) -> "Distance":
+        length = round(self.km / num, 2)
+        return Distance(length)
+
+    def __lt__(self, other: Union["Distance", int, float]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        else:
+            return self.km < other
```

Comment:
```suggestion
        return self.km < other
```

Fix in other places too